### DISCUSSION TITLE
주석 정리하는 것에 대해서 어떤 라벨(태그)를 붙이는 것이 좋을까요?

### DISCUSSION THREAD

**Discussion by hyunzxn** [2022-Harmony/NewsCommunity-bFinal#120] [General]
- comment
- doc
- 기타

---

저는 comment는 뭔가 댓글이랑 겹치는 거 같고 doc는 정말 문서에 대해서 해야된다고 생각하고 있긴 한데 그렇다고 refactor도 좀 이상한 거 같기도 하네요. 어떻게 하면 좋을까여?

**Comment by enyo9rt** [comment]
[커밋 위키](https://github.com/2022-Harmony/NewsCommunity-bFinal/wiki/%EC%BB%A4%EB%B0%8B-%EB%A9%94%EC%8B%9C%EC%A7%80-%ED%98%95%EC%8B%9D)
저는 이 사진 컨벤션 저희 맨처음에 정했던거라 이거보고 하고는잇는데 찬혁님은 doc 쓰는 거 같더라구요

**Reply by hyunzxn** [reply]
음...저기엔 일단 Comment라고 되어있네요. 그러면 Comment로 통일하는것도 괜찮을 거 같네요. doc는 진짜 문서에 붙여야 할 것 같은 느낌이라 ㅋㅋㅋ
